authenticity.cpp

```diff
@@ -189,37 +185,30 @@ std::string Authenticity::Assay
         }
 
     // Validate all data files.
-    fs::path original_path(fs::current_path());
-    if(0 != chdir(data_path.string().c_str()))
+    std::vector<md5sum_for_file> sums;
+    std::string error_message, md5;
```

Comment:
To be checked, but I think lmi sources don't declare multiple variables in the same declaration.